src/state/opamFileTools.ml

```diff
@@ -930,7 +930,7 @@ let t_lint ?check_extra_files ?(check_upstream=false) ?(all=false) t =
               ("file" | "path" | "local" | "rsync") -> true
             | _, _ -> false)
            && (Filename.is_relative u.path
-               || OpamStd.String.contains ~sub:".." u.path))
+               || OpamFilename.might_escape ~sep:`Unix u.path))
```

Comment:
```suggestion
               || OpamFilename.might_escape u.path))
```

Reply:
why ?

Reply:
oops my bad, it's a remnant from an earlier review. I'll remove it